.github/config.yml

```diff
@@ -0,0 +1,28 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
```

Comment:
Can you change the name of the file to something like `welcome_bot.yml`

Reply:
yes sure!